File: MY PROGRAMS/LB/JAVA/NUMBER/a10q4.java
/*	4. Write a program which accept temperature in Fahrenheit and convert it into celsius. (1 celsius = (Fahrenheit -32) * (5/9)).

	Input:	10
	Output:	-12.222222	

	Input:	34
	Output:	1.111111	*/




import java.util.*;

class Assignment10
{
	public double Return(double df)
	{
		double dc=0.0;

		dc=(df-32)*0.555556;

		return(dc);
	}
}

class a10q4
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		double dvalue=0.0,dret=0.0;

		System.out.println("Enter the tempreture in fahrenheit: ");
		dvalue=sobj.nextDouble();

		Assignment10 aobj=new Assignment10();

		dret=aobj.Return(dvalue);

		System.out.println("Conversion of tempreture in fahrenheit into celcius is "+dret);
	}
}

File: MY PROGRAMS/LOGIC BUILDING-13 ASSIGNMENTS/a10q4.c
/* Write a program which accept temperature in Fahrenheit and convert it into celsius. (1 celsius = (Fahrenheit -32) * (5/9))

Input:10
Output:-12.2222

Input:34
Output:1.11111

*/




#include<stdio.h>

double Celcius(double dfar)
{
	double dcel=0.0;
	
	dcel=(dfar-32)*0.555555556;

	return(dcel);
}

int main()
{
	double dvalue=0.0,dret=0.0;

	printf("Enter the Kilometer:");
	scanf("%lf",&dvalue);

	dret=Celcius(dvalue);

	printf("Fahrenheit in celcius %lf",dret);

	return(0);
}

